src/game/physics.cpp

```diff
@@ -1050,7 +1050,7 @@ namespace physics
             { \
                 int col = (int(mcol[2]*mq) + (int(mcol[1]*mq) << 8) + (int(mcol[0]*mq) << 16)); \
                 regularshape(mp, mt, col, 21, 20, mz, mo, ms, 1, 10, 0, 20); \
-                if(mw >= 0) playsound(mw, mo, pl); \
+                if((mw) >= 0) playsound(mw, mo, pl); \
```

Comment:
What warning does this fix?

Reply:
Have you checked the commit? See https://github.com/blue-nebula/base/pull/248/commits/d0818f3992e248851eebf22a6deae86360b73642 and https://github.com/blue-nebula/base/pull/248#discussion_r905838467